### PR TITLE
Add version check for Node 8 binaries

### PR DESCRIPTION
**Pull request by mattcolegate**

### CODE REVIEW

File: extract_all_binaries.js [hunk 1]
```diff
@@ -112,6 +112,9 @@ var getSupportedNodeVersionOrExit = function() {
 	if (process.version.indexOf('v7') === 0) {
 		return '7';
 	}
+	if (process.version.indexOf('v8') === 0) {
```
Comment (gibfahn) [inline extract_all_binaries.js]: You could also just do:

```js
if (process.version.startsWith('v8')) {
```

Comment (mattcolegate) [inline extract_all_binaries.js]: Is that more performant?

Comment (gibfahn) [inline extract_all_binaries.js]: >Is that more performant?

Probably not, just looks clearer.

Even nicer alternative:

```js
const supportedVersions = ['0.10','0.12','2','4','5','6','7','8'];
for (var version of supportedVersions) 
  if (process.version.startsWith(`v${version}.`))
    return version;
```

but it's mostly just cosmetic.

**Comment by tobespc** [comment]
lgtm

**Pull request merged**